src/main/java/org/I0Itec/zkclient/IZkConnection.java

```diff
@@ -38,6 +38,8 @@
     public String create(String path, byte[] data, List<ACL> acl, CreateMode mode) throws KeeperException, InterruptedException;
 
     public void delete(String path) throws InterruptedException, KeeperException;
+    
+    public void delete(String path, int version) throws InterruptedException, KeeperException;
```

Comment:
@granthenke I thought about it, but ended up not doing it because of compatibility. I assumed that there are applications out there using the call without a version and I didn't want to break anything, but otherwise I agree that it works.